vllm/engine/llm_engine.py

```diff
@@ -20,7 +20,7 @@
                            SequenceGroupOutput, SequenceOutput, SequenceStatus)
 from vllm.transformers_utils.tokenizer import (detokenize_incrementally,
                                                TokenizerGroup)
-from vllm.utils import Counter, set_cuda_visible_devices, get_ip, get_open_port, get_distributed_init_method
+from vllm.utils import Counter, set_cuda_visible_devices, get_ip, get_open_port, get_distributed_init_method, is_neuron
```

Comment:
This line looks too long
```suggestion
from vllm.utils import (
Counter, set_cuda_visible_devices, get_ip, get_open_port, get_distributed_init_method, is_neuron)
```
and execute the formatter.